apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -315,4 +333,4 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
     }
 }
 
-export default withStyles(styles)(ScheduleCalendar);
+export default withStyles(styles)(ScheduleCalendar);
```

Comment:
```suggestion
export default withStyles(styles)(ScheduleCalendar);

```

Reply:
It's best-practice to leave a new-line at the end of files. I.e., every line in the file should have new line at the end of it, including the last one.

Reply:
OK! I will add an empty line on the file that has not.